iceoryx_binding_c/include/iceoryx_binding_c/publisher.h

```diff
@@ -40,7 +40,7 @@ typedef struct
     bool offerOnCreate;
 
     /// @brief describes whether a publisher blocks when subscriber queue is full
-    ENUM iox_SubscriberTooSlowPolicy subscriberTooSlowPolicy;
+    ENUM iox_ConsumerTooSlowPolicy subscriberTooSlowPolicy;
```

Comment:
Maybe a short note that this will deprecate within the next releases

Reply:
Not here. The struct member keeps the name to show the semantic meaning in this context. Think of it as not naming an `int64_t` member not `int64`